apps/studio/components/interfaces/Support/DiscordCTACard.tsx

```diff
@@ -0,0 +1,81 @@
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import { AnimatePresence, motion } from 'framer-motion'
+import Link from 'next/link'
+import router from 'next/router'
+import { useEffect, useState } from 'react'
+import SVG from 'react-inlinesvg'
+import { Button } from 'ui'
+import { NO_ORG_MARKER } from './SupportForm.utils'
+
+interface DiscordCTACardProps {
+  organizationSlug?: string | null
+}
+
+export const DiscordCTACard = ({ organizationSlug }: DiscordCTACardProps) => {
+  const { data: organizations } = useOrganizationsQuery()
+  const [isVisible, setIsVisible] = useState(false)
+
+  useEffect(() => {
+    const timer = setTimeout(() => setIsVisible(true), 800)
+    return () => clearTimeout(timer)
+  }, [])
```

Comment:
Just like everything else in this card: I’m following some basic motion styles set out in AIAssistantOption. This helps smooth out layout shift when conditionally-loaded items (e.g. based on plan tier) come through.